moveit_simple_controller_manager/include/moveit_simple_controller_manager/follow_joint_trajectory_controller_handle.h

```diff
@@ -64,9 +63,14 @@ class FollowJointTrajectoryControllerHandle : public ActionBasedControllerHandle
     if (!controller_action_client_)
       return false;
 
+    /*
+     * This should never happen as the moveit_simple_controller_manager now loads
+     * MULTI_DOF_FOLLOW_JOINT_CONTROLLER_HANDLE for multi-dof joints
+     */
     if (!trajectory.multi_dof_joint_trajectory.points.empty())
     {
-      ROS_WARN("FollowJointTrajectoryController: %s cannot execute multi-dof trajectories.", name_.c_str());
+      ROS_ERROR("FollowJointTrajectoryController: %s cannot execute multi-dof trajectories. Check moveit_simple_controller_manager to load MOVEIT_PLUGINS_MULTI_DOF_FOLLOW_JOINT_CONTROLLER_HANDLE correctly", name_.c_str());
+      return false;
```

Comment:
please revert these changes -- we can't apply them to indigo at this point (could change the behavior of existing configurations)